src/domain/swaps/swaps.repository.e2e-spec.ts

```diff
@@ -141,7 +142,9 @@ describe('CowSwap E2E tests', () => {
 
           expect(actual).toEqual({
             ...expectedObject,
-            fullAppData: JSON.parse(expectedObject.fullAppData),
+            fullAppData:
+              typeof expectedObject.fullAppData === 'string' &&
+              JSON.parse(expectedObject.fullAppData as string),
```

Comment:
`fullAppData` can be `false` here if it is not a string. Maybe we should use a ternary here?

Reply:
Good catch! I've simply removed the first part of the condition, I think it's not really useful, wdyt? Which ternary would you propose otherwise? Maybe I'm not getting it.

Reply:
[5118bed](https://github.com/safe-global/safe-client-gateway/pull/1533/commits/5118beddfa2bdb98fd00114e21ae415ecfbc903e)